lib/vagrant-service-manager/command.rb

```diff
@@ -76,29 +84,76 @@ def copy_from_box(hIP, hport, husername, hprivate_key_path, source, destination)
         end
       end
 
-      def execute_docker_info
-        # this execute the operations needed to print the docker env info
+      def check_if_a_service_is_running?(service)
+        command = "systemctl status #{service}"
```

Comment:
@LalatenduMohanty : Sure! For k8s, will need to iterate over multiple services and probably do some more. One of the reason I separated out the method to check for service status. Noted!